eland/series.py

```diff
@@ -1560,6 +1560,22 @@ def nunique(self) -> pd.Series:
         results = super().nunique()
         return results.squeeze()
 
+    def unique(self) -> pd.Series:
+        """
+            Returns unique values in the ascending order
```

Comment:
We should probably document our ordering being different than pandas:

```
Returns all unique values within a Series. Note that behavior is slightly different between pandas and Eland: pandas will return values in the order they're first seen and Eland returns values in sorted order.
```

Reply:
Man! You're writing skills are good :)